### DISCUSSION TITLE
Page references using auto-computed page titles?

### DISCUSSION THREAD

**Discussion by johnkerl** [quarto-dev/quarto-cli#478] [Q&A] [answered]
Suppose I have `foo.md` which has `# Title is Foo` (or, `_quarto.yml` has `href: foo.md` and `text: "Title is Foo"`).

`foo.md`:
```markdown
# This is Foo

Lorem ipsum.
```

Suppose inside `bar.md` I want to have a link to `foo.md` -- using Foo's nice title as the link-display, rather than its markdown filename.

`bar.md`:
```markdown
...

Have:
For more information please see [This is Foo](foo.md)

Want:
For more information please see {some syntax involving "foo.md"}

```

AFAICT I need to 'double-enter' this via `[Title is Foo](foo.md)`. I don't see a syntax which will let me just type some punctuation around the text `foo.md` inside `bar.md` and have Quarto "find" the nice name "Title is Foo".

I do see this for figures, theorems, etc etc at https://quarto.org/docs/authoring/cross-references.html#references but not for pages -- am I missing something?

**Comment by jjallaire** [comment, marked as answer]
No, there isn't currently a mechanism for this. This is partly because unlike figures and tables the text is much less straightforward to produce automatically (sometimes you want less words, different casing, etc.). Nevertheless I do agree it would be good to have something like this for the times when it is useful -- we'll put in on our list.

**Reply by johnkerl** [reply]
Thanks @jjallaire !

And I realize if we really want `sec:` references we can add them to file contents
